src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -6,6 +6,7 @@
 from typing import List, Optional, Type, Union
 
 import numpy as np
+from hdmf.testing import TestCase as HDMFTestCase
```

Comment:
```suggestion
```

You are not using ths I think.

Reply:
Ah, good catch

I had meant to use `assertRaisesWith` (the reason it's `hdmf` specific) as a class method like many `unittest.TestCase` do but for some reason it only operates as an instance method

I've added an `isinstance` assertion and added docstring note about it

Reply:
Yeah, I remember; I really prefer the HDMF implementation so that I don't have to think about how to cast everything into regex

Also makes the test case assertions easier to read